scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/session/log/MultiVideoDataReader.java

```diff
@@ -27,7 +27,20 @@ public MultiVideoDataReader(File dataDirectory, LogProperties logProperties, Bac
          Camera camera = cameras.get(i);
          try
          {
-            VideoDataReader reader = new VideoDataReader(camera, dataDirectory, logProperties.getVideo().getHasTimebase());
+            VideoDataReader reader;
+            if (camera.getType().toString().equals("MAGEWELL_CAPTURE"))
+            {
+               reader = new MagewellVideoDataReader(camera, dataDirectory, logProperties.getVideo().getHasTimebase());
+            }
+            else if (camera.getType().toString().equals("CAPTURE_CARD"))
+            {
+               reader = new BlackMagicVideoDataReader(camera, dataDirectory, logProperties.getVideo().getHasTimebase());
+            }
+            else
+            {  // Older logs won't have the camera type set correctly, if there isn't a type set this as the only option
```

Comment:
If we don't know the type, we will try to load it with the BlackMagicVideoDataReader but no promises